CONTRIBUTING.md

```diff
@@ -46,12 +46,12 @@ Note that different screen sizes must be located in different test suites.
 
 ## Add translation
 
-- Copy the _app/i18n/locales/en-US/_ folder into _app/locales/{id}_ changing `{id}` with the identifier of the translation. See [leptos-fluent languages documentation].
+- Copy the [english](/app/i18n/locales/en-US/) folder into _app/i18n/locales/{id}/_ changing `{id}` with the identifier of the translation. See [leptos-fluent languages documentation].
```

Comment:
Please, would you mind to remove the link here? I think that is confusing for plain Markdown readers because it forces them to navigate through the link to know which path it refers to.

Reply:
Done!